### PR TITLE
feat: method to create new Keybase team or subteam

### PR DESCRIPTION
**Pull request by zackster**

### CODE REVIEW

**Comment by pzduniak** [comment]
will take a look at this tomorrow

File: src/team-client/types.ts [hunk 1]
```diff
@@ -27,6 +31,7 @@ export interface ListTeamMembershipsParam {
 
 export type AddMembersResult = any /* TODO: this */
 export type RemoveMemberResult = any /* TODO: this */
+export type CreateTeamResult = any /* TODO: this */
```
Comment (pzduniak) [inline src/team-client/types.ts]:
```
type TeamCreateResult struct {
	TeamID       TeamID `codec:"teamID" json:"teamID"`
	ChatSent     bool   `codec:"chatSent" json:"chatSent"`
	CreatorAdded bool   `codec:"creatorAdded" json:"creatorAdded"`
}
```

File: lib/team-client/index.d.ts [hunk 2]
```diff
@@ -1,7 +1,16 @@
...
 declare class Team extends ClientBase {
+    /**
+     * Create a new Keybase team or subteam
+     * @memberof Team
+     * @param creation - the name of the team to create
+     * @returns -
+     * @example
+     * bot.team.create({"team": "phoenix"}).then(res => console.log(res))
+     */
+    create(creation: CreateTeamParam): Promise<CreateTeamResult>;
```
Comment (pzduniak) [inline lib/team-client/index.d.ts]: actually... I think we can just accept a string here, at least for now

File: src/team-client/index.ts [hunk 3]
```diff
@@ -8,8 +10,25 @@ import {
   ListTeamMembershipsResult,
 } from './types'
 
-/** The wallet module of your Keybase bot. For more info about the API this module uses, you may want to check out `keybase wallet api`. */
+/** The team module of your Keybase bot. For more info about the API this module uses, you may want to check out `keybase team api`. */
```
Comment (nathunsmitty) [inline src/team-client/index.ts]: 😆

**Comment by pzduniak** [comment]
FWIW this PR is great as-is, I just need to go through the docs lib and fix it per your suggestions. Will do it this week.

**Comment by pzduniak** [comment]
Merged as #178 without the README change. Thanks!

**Pull request closed without merging**